modules/cnf-running-the-performance-creator-profile.adoc

```diff
@@ -0,0 +1,183 @@
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on bare metal hardware.
+* A node with `podman` and OpenShift CLI (`oc`) installed.
+
+
+.Procedure
+
+. Check the machine config pool:
+
++
+[source,terminal]
+----
+$ oc get mcp
+----
+.Example output
++
+[source,terminal]
+----
+NAME         CONFIG                                                 UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master       rendered-master-acd1358917e9f98cbdb599aea622d78b       True      False      False      3              3                   3                     0                      22h
+worker-cnf   rendered-worker-cnf-1d871ac76e1951d32b2fe92369879826   False     True       False      2              1                   1                     0                      22h
+----
+
+. Optional: Display help for the PPC tool:
++
+[source,terminal]
+----
+$ podman run --entrypoint performance-profile-creator quay.io/openshift-kni/performance-addon-operator:4.8-snapshot -h
+----
++
+.Example output
++
+[source,terminal]
+----
+A tool that automates creation of Performance Profiles
+
+Usage:
+  performance-profile-creator [flags]
+
+Flags:
+      --disable-ht                        Disable Hyperthreading
+  -h, --help                              help for performance-profile-creator
+      --mcp-name string                   MCP name corresponding to the target machines (required) (default "worker-cnf")
+      --must-gather-dir-path string       Must gather directory path (default "must-gather")
+      --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
+      --profile-name string               Name of the performance profile to be created (default "performance")
+      --reserved-cpu-count int            Number of reserved CPUs (required)
+      --rt-kernel                         Enable Real Time Kernel (required) (default true)
+      --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
+      --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
+      --user-level-networking             Run with User level Networking(DPDK) enabled
+----
+
+. Run the performance profile creator tool in discovery mode:
++
+[NOTE]
+====
+Discovery mode inspects your cluster using the output from `must-gather` enabling you to have a better understanding of the values of the arguments you can supply to the tool.
+====
++
+[source,terminal]
+----
+$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z quay.io/openshift-kni/performance-addon-operator:4.8-snapshot --info --must-gather-dir-path /must-gather
```

Comment:
This was further updated :) Please refer this commit here: https://github.com/openshift-kni/performance-addon-operators/pull/628/commits/951ec0afcfc8b251fd397a6c5c8bc406ad09eeba

Reply:
I am trying to restructure the section a little but is this a valid way to run the discovery mode 
$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z quay.io/openshift-kni/performance-addon-operator:4.8-snapshot --info log --must-gather-dir-path /must-gather

Reply:
I can confirm that the above command works but I would recommend that we use a generic path here to indicate clearly how the must gather output path must be specified:
```
podman run --entrypoint performance-profile-creator -v /path/to/must-gather-output:/must-gather:z quay.io/openshift-kni/performance-addon-operator:4.8-snapshot --info=log --must-gather-dir-path /must-gather
```

Reply:
This looks good!